galaxy/wrapper/test-data/profiler_result2.tabular

```diff
@@ -0,0 +1,3 @@
+bin labels		-0.0Kb	0.0Kb
```

Comment:
I'll go with option 1 then. I don't really see the point of the argument, it only exists because the arguments are partly shared with `plotProfile` :P